Cargo.toml

```diff
@@ -1,23 +1,19 @@
 [package]
-name = "cargo"
```

Comment:
Yep, exactly right. I didn't realize how much it had affected Cargo.toml. I've fixed and re-pushed.